modules/video/src/video-builder.ts

```diff
@@ -0,0 +1,75 @@
+const VIDEO_BUILDER_OPTIONS = {
+  width: 512,
+  height: 512,
+  framerate: 24,
+  bitrate: 200,
+  realtime: false
+};
+
+type VideoBuilderOptions = Partial<typeof VIDEO_BUILDER_OPTIONS>;
+
+function nextEvent(target, name): Promise<{data: any}> {
+  return new Promise((resolve) => {
+    target.addEventListener(name, resolve, {once: true});
+  });
+}
+
+export default class VideoBuilder {
+  worker: Worker = new Worker('node_modules/webm-wasm/dist/webm-worker.js');
```

Comment:
We typically include such workers in our own dist, rather than import them from installed packages. Given that it is apache licensed there are no concerns with copying it, especially if we give some attribution.

The `libs` folder in each module is the place to put such "binaries". There are a bunch of modules that do this.


The downside is we don't automatically get access to patches but I think it may cause problems for users if we depend on an npm install tree.

Reply:
Done. It seems like they don't update all that regularly, so we should be fine for the time being. 

That said, I'm having trouble with the webpack config here... doesn't seem to be finding the worker code when I try to import (I see there's a rule for `test: /\.worker\.js$/,` but it doesn't seem to be working). From what I can tell from the parquet-wasm and draco loader, it looks like they're loaded from third party CDNs. Any pointers on what to do here?